app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,128 @@
+<div id="myCarousel" class="carousel slide" data-ride="carousel">
+    <!-- Indicators -->
+  <ol class="carousel-indicators">
+    <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+    <li data-target="#myCarousel" data-slide-to="1"></li>
+    <li data-target="#myCarousel" data-slide-to="2"></li>
+    <li data-target="#myCarousel" data-slide-to="3"></li>
+    <li data-target="#myCarousel" data-slide-to="4"></li>
+  </ol>
+
+  <div class="carousel-inner" role="listbox">
+    <div class="item active">
+      <img src="http://www.qualityfootwear.ca/wp-content/uploads/2014/03/park_lane-0079.jpg" alt="Sandals">
```

Comment:
Would it be possible to store this information? Are Sandals a category? If so, having a way to store this would mean that you could re-use it elsewhere on the site if you so chose and update both without having to go to both places to update.